pallets/security/src/default_weights.rs

```diff
@@ -0,0 +1,59 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
Maybe we can add a comment here at the very top of this file that it was created manually? Might be helpful since all other default_weights.rs files are generated automatically.

Something like
```
/// The respective weights were picked manually based on the functions each extrinsic uses.
```